solidity/contracts/converter/ConverterBase.sol

```diff
@@ -446,6 +432,29 @@ abstract contract ConverterBase is ConverterVersion, IConverter, TokenHolder, Co
         emit Conversion(_sourceToken, _targetToken, _trader, _amount, _returnAmount, int256(_feeAmount));
     }
 
+    /**
+     * @dev transfers funds held by the contract and sends them to an account
+     *
+     * @param token ERC20 token contract address
+     * @param to account to receive the new amount
```

Comment:
IMO, we should "redeploy" our previous convention of aligning the descriptions.
It is otherwise unreadable when viewing the contract, as in the example above:
```
@param to account to receive the new amount
```
Keep in mind that some users (including ourselves) read our documentation directly in the contracts and not on our GitBook.